pkg/openevec/pod.go

```diff
@@ -66,80 +66,80 @@ func processVLANs(vlans []string) (map[string]int, error) {
 	return m, nil
 }
 
-func PodDeploy(appLink string, cfg *EdenSetupArgs) error {
+func PodDeploy(appLink, podName, podMetadata, registry string, podNetworks, portPublish, acl, vlans, mount, disks, profiles, appAdapters []string, noHyper bool, vncDisplay uint32, vncPassword, diskSize, volumeSize, appMemory, volumeType string, appCpus uint32, pinCpus bool, imageFormat string, sftpLoad, directLoad, openStackMetadata bool, datastoreOverride string, aclOnlyHost bool, startDelay uint32, cfg *EdenSetupArgs) error {
```

Comment:
Seems you put wrong order here (or on caller side): appMemory should be after volumeType.